docs/installing-solidity.rst

```diff
@@ -179,12 +189,98 @@ Install it using ``brew``:
     # eg. Install 0.4.8
     brew install solidity.rb
 
-Gentoo Linux has an `Ethereum overlay <https://overlays.gentoo.org/#ethereum>`_ that contains a solidity package.
-After the overlay is setup, ``solc`` can be installed in x86_64 architectures by:
+Static Binaries
+===============
 
-.. code-block:: bash
+We maintain a repository containing static builds of past and current compiler versions for all
+supported platforms at `solc-bin`_. It is also the location where you can find the nightly builds.
+
+The repository is not only a quick and easy way for end users to get binaries ready to be used
+out-of-the-box but it is also meant to be friendly to third-party tools:
+
+- The content is mirrored to https://solc-bin.ethereum.org where it can be easily downloaded over
+  HTTPS without any authentication, rate limiting or the need to use git.
+- Content is served with correct `Content-Type` headers and lenient CORS configuration so that it
+  can be directly loaded by tools running in the browser.
+- Binaries do not require installation or unpacking (with the exception of older Windows builds
+  bundled with necessary DLLs).
+- We strive for a high level of backwards-compatibility. Files, once added, are not removed or moved
+  without providing a symlink/redirect at the old location. They are also never modified
+  in place and should always match the original checksum. The only exception would be broken or
+  unusable files with a potential to cause more harm than good if left as is.
+
+The same binaries are in most cases available on the `Solidity release page on github`_. The
+difference is that we do not generally update old releases. This means that we do not rename them if
+the naming convention changes and we do not add builds for platforms that were not supported at the
+time of release. We do it only in ``solc-bin``.
+
+``solc-bin`` repository contains several top-level directories, each representing a single platform.
+Each one contains a ``list.json`` file listing the available binaries. For example in
+``emscripten-wasm32/list.json`` you will find the following information about version 0.7.4:
+
+.. code-block:: json
+
+    {
+      "path": "solc-emscripten-wasm32-v0.7.4+commit.3f05b770.js",
+      "version": "0.7.4",
+      "build": "commit.3f05b770",
+      "longVersion": "0.7.4+commit.3f05b770",
+      "keccak256": "0x300330ecd127756b824aa13e843cb1f43c473cb22eaf3750d5fb9c99279af8c3",
```

Comment:
Hmm, didn't we also had a sh256 hash next to this? That is way more useful for anyone downloading a static compiler from here and trying to verify it on CI (as that works with `shasum -a 256 <file>`).

Reply:
Apparently not. It's definitely not something that was removed recently and I don't see it being there in older revisions of `list.json` either.

There's [sha3sum](https://github.com/maandree/sha3sum) which provides `keccak-256sum` and is available as a package  at least on Arch, Ubuntu and Homebrew, likely on other systems too. I'll mention it.

But yeah, `sha256sum` is in `coreutils` so definitely more common.

Reply:
Do we want this? Should I create an issue?

Reply:
I'd vote for having sha256 as it is ubiquitous, but it shouldn't be a blocker for merging this PR.

Reply:
Reported in #10379 so I'm marking it as resolved.